### DISCUSSION TITLE
Work with Inventory/Item Names

### DISCUSSION THREAD

**Discussion by Art0sh** [ORelio/Minecraft-Console-Client#1894] [Q&A]
Hey, so I was wondering if it would be possible to maybe work with item names in scripts. I didnt find anything in /inventory help, but is it somehow possible? I wanted to get the item name of a special item (which location is locked in a plugin inventory gui) to remotely get information about the skill level of my bot, so I dont have to kick the bot, login with my mc client, look it up and then restart the bot...

**Comment by ORelio** [comment]
If you manipulate the inventory from a C# script, you can access properties of each item ([Item.cs](https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/Inventory/Item.cs)).
In `/inventory list`, the item display name should be displayed; but not the lore (would take too much visual space).
See also: #1905
